packages/adapter-cloudflare-workers/package.json

```diff
@@ -49,6 +49,6 @@
 	},
 	"peerDependencies": {
 		"@sveltejs/kit": "^2.0.0",
-		"wrangler": "^3.91.0"
+		"wrangler": ">=3.0.0"
```

Comment:
this will satisfy 5 and above, is that what we want?

Reply:
good point. updated